src/get_games.py

```diff
@@ -88,13 +86,14 @@ def download_games(name: str,
 
     user = lichess.api.user(name)
 
-    token = setup()
-
     # print(user['perfs'].values())
     # for i in user['perfs']:
     #    print(i, "\t", user['perfs'][i])
 
     # time in milliseconds since Jan 1st 1970
+    if time_period is None:
+        time_period = []
+    initial_time, latest_time = time_period
```

Comment:
Errors out if time_period was originally None.

Reply:
Is this better or do you think a different pattern would be better?
I don't really like how these ifs are but...

Reply:
Using tuples simplify a bit the logic here.